src/logit_logger.cpp

```diff
@@ -39,6 +39,11 @@ static void logFromOpen62541 (
 		case UA_LOGLEVEL_WARNING: logItLogLevel = Log::WRN; break;
 		default: logItLogLevel = Log::ERR; // just in case they added a new level.
 	}
+
+	Log::LOG_LEVEL userLogLevel;
+	Log::getComponentLogLevel(Log::getComponentHandle("open62541"), userLogLevel);
```

Comment:
This might fail (e.g. component nor registered). Note that getComponentLogLevel returns a bool and you have userLogLevel uninitialized which will cause indeterministic behaviour when component was forgotten to be registered (or was not registered on purpose). Please improve it.

Reply:
I am merely using getComponentHandle to get the handle. What guarantees that the component is defined is that it is initialized in UaSession::connect().
I think that an additional check here is redundant, on the other hand it doesn't cost much to add it.

Reply:
Added